front/src/applications/operationalStudies/components/Scenario/ScenarioContent.tsx

```diff
@@ -92,33 +87,30 @@ const ScenarioContent = ({
   }, [scenario, isMacro]);
```

Comment:
You can obtain the same result by adding `trainSchedules` in dependencies (use can fix typo un useEffect please `doImport` -> `dtoImport`),